tests/tck/features/MatchAcceptance2.feature

```diff
@@ -1316,7 +1316,7 @@ Feature: MatchAcceptance2
       WITH n.num AS n2
       RETURN n2.num
       """
-    Then a TypeError should be raised at runtime: PropertyAccessOnNonMap
+    Then a TypeError should be raised at runtime: InvalidArgumentValue
```

Comment:
I'd rather change our definition of the `PropertyAccessOnNonMap` step instead of modifying the TCK tests.

Reply:
👍